setup.py

```diff
@@ -47,7 +47,7 @@ def get_packages(package):
     author_email="tom@tomchristie.com",
     packages=get_packages("morcilla"),
     package_data={"morcilla": ["py.typed"]},
-    install_requires=["sqlalchemy>=1.3,<1.5"],
+    install_requires=["sqlalchemy>=1.3,<1.4.42"],
```

Comment:
why 1.4.42?

Reply:
Library (also upstream) is broken with that version https://github.com/encode/databases/issues/512.